sktime/benchmarking/_lib_mini_kotsu/run.py

```diff
@@ -58,16 +61,42 @@ def run(
         results_df = pd.DataFrame(columns=["validation_id", "model_id", "runtime_secs"])
         results_df["runtime_secs"] = results_df["runtime_secs"].astype(int)
 
+    tqdm_available = False
+    if verbose:
+        try:
```

Comment:
try/except is not good style, if you know the condition you want to check.
In that case, simply check the condition, you can do this with `_check_soft_dependencies` as I have recommended, kindly search the code base.